.travis.sh

```diff
@@ -0,0 +1,88 @@
+#!/bin/bash
+
+# https://github.com/travis-ci/travis-rubies
+headline() {
+    echo -e "\033[33;1m${1}\033[0m"
+}
+
+fold_wrap() {
+    local header=$1; shift
+    local cmd=$1; shift
+    
+    travis_fold start $cmd
+    headline "$header"
+    $cmd
+    local result=$?
+    travis_fold end $cmd
+    return $result
+}
+
+### BEFORE_INSTALL ###
+
+install_r_packages() (
+    set -ex
+    
+    # Cairo was missing on OSX, no idea why this wasn't installed :-( Needed for repr
+    if [[ "$TRAVIS_OS_NAME" == osx ]]; then
+        Rscript -e "install.packages('Cairo')"
+    fi
+    
+    # Also test newer versions so that we catch errors earlier
+    if [[ "$DEPS" == github ]]; then
+        Rscript -e 'devtools::install_github("snoweye/pbdZMQ", "irkernel/repr", "irkernel/IRdisplay")'
+    fi
+)
+
+install_miniconda() (
+    set -ex
+    
+    # http://conda.pydata.org/docs/travis.html
+    if [[ "$TRAVIS_OS_NAME" == osx ]]; then os=MacOSX; else os=Linux; fi
+    wget "https://repo.continuum.io/miniconda/Miniconda3-latest-$os-x86_64.sh" -O miniconda.sh
+    bash miniconda.sh -b -p "$HOME/miniconda"
+    rm miniconda.sh # needed to prevent a error in the R CMD check part
+)
+
+configure_conda() (
+    set -ex
+    
+    hash -r
+    conda config --set always_yes yes --set changeps1 no
+    conda update -q conda
```

Comment:
IMO these two belong together in a "create_conda_environment". If there is a split then IMO between `conda info` and `conda create` (e.g. miniconda is not always useable without the `conda update conda` part)